ipaplatform/debian/paths.py

```diff
@@ -17,11 +17,11 @@
 
 class DebianPathNamespace(BasePathNamespace):
     BIN_HOSTNAMECTL = "/usr/bin/hostnamectl"
+    FIPS_MODE_SETUP = "/bin/fips-mode-setup"
+    SYSTEMD_TMPFILES = "/bin/systemd-tmpfiles"
```

Comment:
It's safe to fix the path in ``ipaplatform/base/paths.py`` instead. Fedora-based systems have a unified ``/usr``.

Reply:
yeah I thought about that.. will amend